app/src/main/java/org/wikipedia/page/PageFragment.java

```diff
@@ -489,6 +489,7 @@ public void onPageFinished(WebView view, String url) {
                     // page has now loaded and trigger the remaining logic ourselves.
                     if (!"true".equals(pcsExists)) {
                         onPageSetupEvent();
+                        bridge.onMetadataReady();
```

Comment:
This call is necessary for the MobileWeb shim javascript code to be executed when the page loads. For some reason this was missing in the latest code.